src/runtime/driver/mod.rs

```diff
@@ -82,7 +83,7 @@ impl Driver {
                     return Ok(());
                 }
                 Err(ref e) if e.raw_os_error() == Some(libc::EBUSY) => {
-                    self.tick();
+                    self.dispatch_completions();
```

Comment:
How would you feel about moving this `submit` function above the earlier `dispatch` function? In some ways, it would come first since it would be used first.

Reply:
Sure, I don't really mind the placement currently.

Reply:
Done.